2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Configure stats pid to 23250
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/.config/wandb/settings
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/Thesis/wandb/settings
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'panda_v2.py', 'program': 'panda_v2.py'}
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():507] Logging user logs to /home/user/Thesis/wandb/run-20230802_144202-uebxgo11/logs/debug.log
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():508] Logging internal logs to /home/user/Thesis/wandb/run-20230802_144202-uebxgo11/logs/debug-internal.log
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:init():547] calling init triggers
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:init():595] starting backend
2023-08-02 14:42:02,130 INFO    Dummy-2   :23250 [wandb_init.py:init():599] setting up manager
2023-08-02 14:42:02,131 INFO    Dummy-2   :23250 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-08-02 14:42:02,132 INFO    Dummy-2   :23250 [wandb_init.py:init():605] backend started and connected
2023-08-02 14:42:02,135 INFO    Dummy-2   :23250 [wandb_init.py:init():695] updated telemetry
2023-08-02 14:42:02,158 INFO    Dummy-2   :23250 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-08-02 14:42:02,787 INFO    Dummy-2   :23250 [wandb_run.py:_on_init():2176] communicating current version
2023-08-02 14:42:02,980 INFO    Dummy-2   :23250 [wandb_run.py:_on_init():2185] got version response upgrade_message: "wandb version 0.15.8 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-08-02 14:42:02,980 INFO    Dummy-2   :23250 [wandb_init.py:init():782] starting run threads in backend
2023-08-02 14:42:03,064 INFO    Dummy-2   :23250 [wandb_run.py:_console_start():2157] atexit reg
2023-08-02 14:42:03,064 INFO    Dummy-2   :23250 [wandb_run.py:_redirect():2012] redirect: SettingsConsole.WRAP_RAW
2023-08-02 14:42:03,064 INFO    Dummy-2   :23250 [wandb_run.py:_redirect():2077] Wrapping output streams.
2023-08-02 14:42:03,064 INFO    Dummy-2   :23250 [wandb_run.py:_redirect():2102] Redirects installed.
2023-08-02 14:42:03,065 INFO    Dummy-2   :23250 [wandb_init.py:init():824] run started, returning control to user process
2023-08-02 14:42:08,017 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Configure stats pid to 23250
2023-08-02 14:42:08,017 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/.config/wandb/settings
2023-08-02 14:42:08,017 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/Thesis/wandb/settings
2023-08-02 14:42:08,017 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'panda_v2.py', 'program': 'panda_v2.py'}
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():507] Logging user logs to /home/user/Thesis/wandb/run-20230802_144208-iijw4o5h/logs/debug.log
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():508] Logging internal logs to /home/user/Thesis/wandb/run-20230802_144208-iijw4o5h/logs/debug-internal.log
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_init.py:init():547] calling init triggers
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-08-02 14:42:08,018 INFO    Dummy-2   :23250 [wandb_init.py:init():590] wandb.init() called when a run is still active
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Configure stats pid to 23250
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/.config/wandb/settings
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from /home/user/Thesis/wandb/settings
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'panda_v2.py', 'program': 'panda_v2.py'}
2023-08-02 14:42:23,793 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():507] Logging user logs to /home/user/Thesis/wandb/run-20230802_144223-35bj2c9g/logs/debug.log
2023-08-02 14:42:23,794 INFO    Dummy-2   :23250 [wandb_init.py:_log_setup():508] Logging internal logs to /home/user/Thesis/wandb/run-20230802_144223-35bj2c9g/logs/debug-internal.log
2023-08-02 14:42:23,794 INFO    Dummy-2   :23250 [wandb_init.py:init():547] calling init triggers
2023-08-02 14:42:23,794 INFO    Dummy-2   :23250 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-08-02 14:42:23,794 INFO    Dummy-2   :23250 [wandb_init.py:init():590] wandb.init() called when a run is still active
2023-08-02 14:42:47,506 WARNING MsgRouterThr:23250 [router.py:message_loop():77] message_loop has been closed
